ultimatejobweb/API_FB.py

```diff
@@ -0,0 +1,44 @@
+# Import and load necessary lib
+import requests
+from bs4 import BeautifulSoup as bs
+
+
+class Manipulation:
+    def extract_jobs_list(self):
+        fb_url = "https://www.facebook.com"
+        jobs_links = []
+        jobs_list = self.find("div", {"class": "_8tk7"})
+        jobs = jobs_list.find_all("a", {"class": "_8sef"})
+        links = [link['href'] for link in jobs]
+        jobs_links = links
+        for link in jobs_links:
+            print(fb_url+link)
+            # Need to push to a db
```

Comment:
Please return value from this method
It will ease testing right now for your methods